src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1473,6 +1473,7 @@ PluginManager.pluginDesc.Freetalk=Experimental anonymous forum. Currently has se
 PluginManager.pluginDesc.WebOfTrust=Allows you to create an anonymous identity. These identities can trust each other, or not, and hence avoid spam and DoS attacks. Backend for chat and social applications over Freenet.
 PluginManager.pluginDesc.WebOfTrustTesting=Very new versions of the Web of Trust plugin, for testing purposes. Compiled by developer "xor". ATTENTION: These are not compiled on a high security computer. There is no key revocation mechanism for their update USK - if it gets hacked, you are in trouble. Updates will be installed at every restart of Freenet, even if auto-updating is disabled in configuration! Can only be fetched over Freenet, not from our central servers. Please unload WebOfTrust first before trying to load this plugin.
 PluginManager.pluginDesc.FlogHelper=Easy to use tool for creating a blog on Freenet. Uses WebOfTrust to automatically announce your new flog so it should show up on the freesite indexes.
+PluginManager.pluginDesc.Sharesite=Tool to publish and update single-page sites based on ShareWiki.
```

Comment:
Maybe simply

> Publish and update single-page websites. Based on ShareWiki.

?

I just realized that our descriptions of plugins do not follow a consistent pattern. UPnP vs. FlogHelper vs. WebOfTrust — very different ways to describe a plugin. But that’s a fix for another day…